catalog/docs/src/main/resources/IntContents.adoc

```diff
@@ -192,23 +201,365 @@ Transforms also work on read operations for metacards in federated sources.
 \http://<DISTRIBUTION_HOST>:<DISTRIBUTION_PORT>/services/catalog/sources/<sourceId>/<metacardId>?transform=<TRANSFORMER_ID>
 ====
 
-===== Metacard Transforms Available in DDF
+===== Metacard Transforms Available in {branding}
 
-Unable to render {children}. Page not found: Included Metacard Transformers.
+DDF includes the following Metacard Transformers:
+
+HTML Metacard Transformer:: transforms a Metacard into an HTML formatted document
+XML Metacard Transformer:: transforms a Metacard into an XML formatted document
+GeoJSON Metacard Transformer:: transforms a Metacard into GeoJSON text
+Thumbnail Metacard Transformer:: retrieves the thumbnail bytes of a Metacard
+Metadata Metacard Transformer:: returns the Metacard.METADATA attribute value when given a Metacard.
+Resource Metacard Transformer:: retrieves the resource bytes of a Metacard product
 
 [NOTE]
 ====
-MetacardTransformers can be added to the system at any time. This endpoint can make use of any registered MetacardTransformers.
+MetacardTransformers can be added to the system at any time.
+This endpoint can make use of any registered `MetacardTransformers`.
 ====
-===== InputTransformers 
 
-This REST Endpoint uses InputTransformers to create metacards from a `create` or a `HTTP POST` operation. The REST Endpoint dynamically finds InputTransformers that support the stated in the HTTP header of a `HTTP POST`. InputTransformers register as Services with a list of Content-Type mime-types. The REST Endpoint receives a list of InputTransformers that match the Content-Type and one-by-one calls the InputTransformers until a transformer is successful and creates a Metacard. For instance, if GeoJSON was in the body of the `HTTP POST`, then the HTTP header would need to include application/json in order to match the mime-type GeoJSON Input Transformer supports.
+====== HTML Metacard Transformer
+
+*Description*
+
+The HTML Metacard Transformer is responsible for translating a Metacard into an HTML formatted document.
+
+*Usage*
+
+Using the REST Endpoint, for example, request a metacard with the transform option set to the HTML shortname.
+
+[source,http]
+----
+http://localhost:8181/services/catalog/0123456789abcdef0123456789abcdef?transform=html
```

Comment:
In a separate ticket, we might want to create a variable for default port, like we did for branding.